packages/cactus-test-tooling/src/main/typescript/fabric/fabric-test-ledger-v1.ts

```diff
@@ -36,6 +36,7 @@ export interface IFabricTestLedgerV1ConstructorOptions {
   envVars?: Map<string, string>;
   logLevel?: LogLevelDesc;
   emitContainerLogs?: boolean;
+  stateDatabase?: string;
```

Comment:
@RafaelAPB Is the `stateDatabase` argument somehow needed for the `getConnectionProfileOrgX` method? (I did a quick search but couldn't find a reference for it so I wasn't sure what it is used for)